### PR TITLE
Publish CRD openapi

### PR DESCRIPTION
**Pull request by soltysh**
/assign @deads2k @sttts

### CODE REVIEW

**Comment by deads2k** [comment]
I think I get it.  mostly net new.

/lgtm

**Comment by deads2k** [comment]
aws resources out

/retest

**Comment by deads2k** [comment]
aws-serial is good except for a known flake

File: pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/flags.go [hunk 1]
```diff
@@ -148,6 +148,8 @@ func ConfigToFlags(kubeAPIServerConfig *kubecontrolplanev1.KubeAPIServerConfig)
 	configflags.SetIfUnset(args, "tls-sni-cert-key", sniCertKeys(kubeAPIServerConfig.ServingInfo.NamedCertificates)...)
 	configflags.SetIfUnset(args, "secure-port", portString)
 
+	configflags.SetIfUnset(args, "feature-gates", "CustomResourcePublishOpenAPI=true")
```
Comment (sttts) [inline pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/flags.go]: this is not good. It means that `CustomResourcePublishOpenAPI` is disabled as soon as another actor (user, operator) enables/disables feature gates.

Comment (sttts) [reply pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/flags.go]: Those flags must be merged, probably just by appending the strings (first this default, then the user provides gates; the later overriding the former).

**Comment by sttts** [comment]
Remove the last commit. We will move the feature gate enablement into the operator.

**Comment by soltysh** [comment]
Current errors are due to other CRDs present in the cluster having problems with their schema. The list looks like:

**Comment by deads2k** [comment]
https://openshift-gce-devel.appspot.com/pr/22045 indicates success on be76fbbc

**Pull request merged**